packages/plugin-essentials/sources/commands/set/version.ts

```diff
@@ -68,6 +70,10 @@ export default class SetVersionCommand extends BaseCommand {
     ]],
   });
 
+  useYarnPath = Option.Boolean(`--yarn-path`, false, {
+    description: `Set the yarnPath setting even if the version can be accessed by Corepack`,
+  });
```

Comment:
We should default this to true if the user doesn't have Corepack enabled (`process.env.COREPACK_ROOT`)